doc/concierge.rst

```diff
@@ -0,0 +1,73 @@
+.. _concierge:
+
+The Ibex Concierge
+==================
+
+.. figure:: https://upload.wikimedia.org/wikipedia/commons/8/80/France_in_XXI_Century._Concierge.jpg
+
+The Ibex Concierge is the friendly caretaker of the Ibex project.
+It's a rotating duty shared by experienced contributors to help newcomers find their way around the project, and to stay on top of the various small tasks necessary to keep the project going.
+
+The Ibex CPU project is a reasonably large open source project.
+Like all projects we experience two challenges:
+we want to lend a helping hand to new developers, answering their questions or helping them with code contributions.
+And we need to stay on top of our "caretaker" tasks, like fixing problems with our continuous integration setup, triaging issues and pull requests, etc.
+The Ibex Concierge combines these two duties in one person.
+
+Please reach out to the Ibex Concierge if you have trouble finding your way around the Ibex project.
+You can find today's Ibex Concierge in the calendar below.
+
+
+Who is Ibex Concierge today?
+----------------------------
+
+The concierge duties rotate between several core developers on a weekly basis.
+You can find today's concierge on duty in a `public calendar <https://calendar.google.com/calendar/embed?src=lowrisc.org_s0pdodkddnggdp40jusjij27h4%40group.calendar.google.com>`_.
+
+* Greg Chadwick (`@GregAC <https://github.com/gregac>`_)
+* Tom Roberts (`@tomroberts-lowrisc <https://github.com/tomroberts-lowrisc>`_)
+* Rupert Swarbrick (`@rswarbrick <https://github.com/rswarbrick>`_)
+* Pirmin Vogel (`@vogelpi <https://github.com/vogelpi>`_)
+* Philipp Wagner (`@imphil <https://github.com/imphil>`_)
+
+You can be Ibex Concierge, too.
+Please talk to any of the current concierges to discuss!
+
+.. raw:: html
+
+    <iframe src="https://calendar.google.com/calendar/embed?src=lowrisc.org_s0pdodkddnggdp40jusjij27h4%40group.calendar.google.com" style="border: 0" width="100%" height="600" frameborder="0" scrolling="no"></iframe>
+
+
+Ibex Concierge duties
+---------------------
+
+The Ibex Concierge is aware of what's happening in the Ibex project, and ensures that everyone feels welcome and is able to work productively.
```

Comment:
Maybe "helps to ensure that everyone feels welcome..." given it's a shared responsibility by all contributors.